apps/api/src/app/user-sessions/user-sessions.service.ts

```diff
@@ -148,7 +148,10 @@ export class UserSessionsService {
       throw new ForbiddenException();
     }
 
-    await this.userSessionsRepository.remove(userSession);
+    await this.anonymizeUserSession(id, user);
+
+    await this.userSessionsRepository.softRemove(userSession);
+
     userSession.id = id;
```

Comment:
is this mapping still needed? soft remove might not remove the id like remove does.